src/slack.coffee

```diff
@@ -242,4 +245,29 @@ class SlackBot extends Adapter
     channel = @client.getChannelGroupOrDMByName envelope.room
     channel.setTopic strings.join "\n"
 
+  customMessage: (data) =>
+    channel = @client.getChannelGroupOrDMByName data.message.envelope.room
+
+    msg = []
```

Comment:
Should this be an object instead of an array?  Below `msg.attachments` is set, which actually does work as "expected" even though it's an array (because arrays are objects) but it is pretty unusual.